Legion/EyeOfAzshara/Trash.lua

```diff
@@ -101,3 +115,7 @@ end
 function mod:AquaSpout(args)
 	self:Message(args.spellId, "Attention", "Alarm", CL.casting:format(args.spellName))
 end
+
+function mod:PolymorphFish(args)
+	self:TargetMessage(args.spellId, args.destName, "Attention", "Info")

```

Comment:
This will only play the sound of for the target itself if it is a healer.

Better solution:
- Remove the `"HEALER"` flag
- Wrap this target message in a `if self:Dispeller("magic") or self:Me(args.destGUID) then` and add `nil, nil, true` to the arguments